pkg/controller/knativeserving/openshift/openshift.go

```diff
@@ -50,7 +55,7 @@ var (
 	extension = common.Extension{
 		Transformers: []mf.Transformer{ingress, egress, deploymentController, annotateAutoscalerService, augmentAutoscalerDeployment, addCaBundleToApiservice, configureLogURLTemplate},
 		PreInstalls:  []common.Extender{addUsersToSCCs, installNetworkPolicies, ensureMaistra, caBundleConfigMap},
-		PostInstalls: []common.Extender{ensureOpenshiftIngress, installServiceMonitor},
+		PostInstalls: []common.Extender{ensureOpenshiftIngress, installServiceMonitor, exposeMetricToServiceMonitor},
```

Comment:
We should change the function names `installServiceMonitor` and/or `exposeMetricToServiceMonitor`? These two functions do similar things, but the names look inconsistent.

Reply:
two function do the same thing but for different namespace.
I think its good to change the function names and refactor code according to that WDYT?

Reply:
Yes, I agree :+1: